ChangeLog

```diff
@@ -1,3 +1,5 @@
+2015-02-23 Gabriel Pratt <gpratt@ucsd.edu>
```

Comment:
Thanks @gpratt for adding the ChangeLog entry. It'll need to match the formatting of the others: two spaces between the date and your name and two spaces between your name and your email address.